sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -2435,99 +2447,105 @@ class CollationSQLExpressionsSuite
   }
 
   test("min_by supports collation") {
-    val collation = "UNICODE"
-    val query = s"SELECT min_by(x, y) FROM VALUES ('a', 10), ('b', 50), ('c', 20) AS tab(x, y);"
-    withSQLConf(SqlApiConf.DEFAULT_COLLATION -> collation) {
-      checkAnswer(
-        sql(query),
-        Seq(
-          Row("a")
+    Seq("UNICODE", "SR", "SR_CI", "SR_AI", "SR_CI_AI").foreach(collation => {
```

Comment:
if we're re-using this exact collation list across multiple tests within the suite, let's separate it out

please see `CollationSupportSuite` for example:
```
  private final String[] testSupportedCollations =
    {"UTF8_BINARY", "UTF8_LCASE", "UNICODE", "UNICODE_CI"};
```

Reply:
Makes sense, fixed this, thnx!